File: C-Python_Solving_50_Problems/12-Max_Number.py
# Python program to find the maximum of two numbers
# take inputs
num1 = float(input('Enter first number: '))
num2 = float(input('Enter second number: '))

# find maximum
max_num = num1 if num1 > num2 else num2

# display result
print('The maximum of {0} and {1} is {2}'.format(num1, num2, max_num))

File: C-Python_Solving_50_Problems/12-Max_Number.c
#include <stdio.h>

int main() {
    float num1, num2, max_num;

    // take inputs
    printf("Enter first number: ");
    scanf("%f", &num1);
    printf("Enter second number: ");
    scanf("%f", &num2);

    // find maximum
    max_num = num1 > num2 ? num1 : num2;

    // display result
    printf("The maximum of %.2f and %.2f is %.2f", num1, num2, max_num);

    return 0;
}
